.github/workflows/e2e-test.yml

```
@@ -256,4 +296,7 @@ jobs:
           default_namespace: ${{ steps.configure.outputs.NAMESPACE }}
           python_version: ${{ env.PYTHON_VERSION }}
           user_namespace: ${{ env.USER_NAMESPACE }}
-          report_name: "E2EMultiUserTests_K8s=${{ matrix.k8s_version }}_cacheEnabled=${{ matrix.cache_enabled }}_multiUser=${{ matrix.multi_user }}"
+          report_name: "E2EMultiUserTests_K8s=${{ matrix.k8s_version }}_cacheEnabled=${{ matrix.cache_enabled }}_multiUser=${{ matrix.multi_user }}_storage=${{ matrix.storage }}_artifactProxy=${{ matrix.artifact_proxy }}"
```

actionlint (1.7.11)

[error] property "storage" is not defined in object type {artifact_proxy: bool; cache_enabled: bool; k8s_version: string; multi_user: bool; test_label: string}

(expression)